tests/functional/harris/harris_2d_100_x_100.py

```
@@ -13,40 +12,48 @@
+# cpp = cpp_lib()
+# startMPI()
 
-cells = (800, 800)
+cells = (100, 100)
+time_steps = 1000
 time_step = 0.005
-final_time = 50
-timestamps = np.arange(0, final_time + time_step, final_time / 5)
+final_time = time_step * time_steps
 
 if cpp.mpi_rank() == 0:
```

Ruff (0.8.2)

Undefined name `cpp`

(F821)